resources/eventing/profile-production.yaml

```diff
@@ -13,19 +13,20 @@ controller:
     maxMessages: -1
   resources:
     limits:
-      cpu: 1000m
-      memory: 512Mi
+      cpu: 2000m
```

Comment:
Are you trying to move these [values](https://github.tools.sap/kyma/management-plane-config/blob/main/resources/runtimes/2.15/base/eventing.yaml)?

Reply:
@mfaizanse Yes, but we want to increase those values also. That's why here cpu limit is bit more than the override in management plane